packages/subgraph/src/utils/tokens/erc1155.ts

```diff
@@ -0,0 +1,353 @@
+import {Address, BigInt, Bytes, ethereum} from '@graphprotocol/graph-ts';
+import {
+  ERC1155Balance,
+  ERC1155Contract,
+  ERC1155TokenIdBalance,
+  ERC1155Transfer
+} from '../../../generated/schema';
+import {ERC1155} from '../../../generated/templates/DaoTemplateV1_0_0/ERC1155';
+import {supportsInterface} from '../erc165';
+import {
+  DECODE_OFFSET,
+  ERC1155_safeBatchTransferFrom,
+  ERC1155_safeTransferFrom,
+  TransferType,
+  getTransferId
+} from './common';
+
+export function supportsERC1155(token: Address): bool {
+  // Double check that it's ERC1155 by calling supportsInterface checks.
+  let erc1155 = ERC1155.bind(token);
+  let introspection_01ffc9a7 = supportsInterface(erc1155, '01ffc9a7'); // ERC165
```

Comment:
should we use the name `introspection_ERC165` instead of `introspection_01ffc9a7` ?